src/Scratchy/source.extension.vsixmanifest

```diff
@@ -8,21 +8,21 @@
         <License>LICENSE.txt</License>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,17.9]">
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,17.10]">
```

Comment:
Note that it's also possible to include all 17.x versions by specifying the version range like so: `[17.0,18.0)` (up to but not including 18.0).